src/libraries/System.ComponentModel.TypeConverter/tests/TrimmingTests/System.ComponentModel.TypeConverter.TrimmingTests.proj

```diff
@@ -1,5 +1,12 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
-
+  <ItemGroup>
+    <TestConsoleAppSourceFiles Include="DesignTimeLicenseContextSerializer.cs">
```

Comment:
There's a trap in the trimming tests that you just fell into :(.

If you don't define any `TestConsoleAppSourceFiles`, it globs all the `*.cs` files and uses them. But as soon as you define a single `TestConsoleAppSourceFiles`, it stops globbing, and so you don't use the other test files in the directory anymore.

You will need to explicitly put all the existing .cs files here in order to get out of the trap.

@joperezr

Reply:
Or this could be done like: 

```xml
<TestConsoleAppSourceFiles Include="DesignTimeLicenseContextSerializer.cs">
   ...
</TestConsoleAppSourceFiles>
<TestConsoleAppSourceFiles Include="*.cs" />
```

Would that work?

Reply:
He can test but I don't think so. Items may contain duplicates, specially when the metadata is different and I'm not sure if the trimming infra is doing any de-duping logic.

Reply:
Ah, I missed the `Exclude`...